README.md

```diff
@@ -108,71 +131,85 @@ Continents are considered `common`.
 
 `102312307`
 
-Basically places that issue passports, notwithstanding the details (like
-empires which actually issue the passports...)
+Basically places that issue passports, notwithstanding the details (like empires which actually issue the passports...)
 
 Continents are considered `common`.
 
 ### county
 
-This needs a better - that is more abstract - name. Like "region" instead of
-state, province, whatever... but for counties.
+`102312313`
 
-Counties are considered `common optional`.
+This needs a better - that is more abstract - name. Like "region" instead of state, province, whatever... but for counties.
+
+Counties are considered `common_optional`.
 
 ### dependency
 
 `102312313`
 
-It's not a sub-region of a country but rather dependent on a parent country for
```

Comment:
NOTE: In some areas `country` and `dependency` are used interchangeably.

Reply:
Seems like every implementation of WOF gets it confused that both dependency and country are needed (country alone won’t suffice). Is there a better way of calling that out?